src/main/resources/mappings/opendistro-ism-config.json

```diff
@@ -0,0 +1,321 @@
+{
+  "_doc": {
+    "properties": {
+      "dynamic": "strict",
+      "policy": {
+        "properties": {
+          "name": {
+            "type": "text",
+            "fields": {
+              "keyword": {
+                "type": "keyword",
+                "ignore_above": 256
+              }
+            }
+          },
+          "schema_version": {
+            "type": "long"
+          },
+          "last_updated_time": {
+            "type": "date",
+            "format": "strict_date_time||epoch_millis"
+          },
+          "default_state": {
+            "type": "keyword"
+          },
+          "states": {
+            "type": "nested",
+            "properties": {
+              "name": {
+                "type": "keyword"
+              },
+              "actions": {
+                "type": "nested",
+                "properties": {
+                  "timeout": {
+                    "type": "text"
```

Comment:
I think we can either convert them to `keyword` or tell ES to skip analyzing.

Reply:
Changed to keyword